proto/dataplane.proto

```diff
@@ -103,24 +132,71 @@ message RouteMap {
   string action = 3;
   uint32 sequence = 4;
 }
-
 message RouterConfig {
   string asn = 1;
   string router_id = 2;
   repeated BgpNeighbor neighbors = 3;
-  repeated BgpAddressFamilyOptions options = 4;
+  optional BgpAddressFamilyIPv4 ipv4_unicast = 4;
+  optional BgpAddressFamilyIPv6 ipv6_unicast = 5;
+  optional BgpAddressFamilyL2vpnEvpn l2vpn_evpn = 6;
   repeated RouteMap route_maps = 5;
 }
-
 message VRF {
   string name = 1;
-  RouterConfig router = 2;
-  VPC vpc = 3;
+  repeated Interface interfaces = 2;
+  optional RouterConfig router = 3;
+}
+message Underlay {
+  repeated VRF vrf = 1;
+}
+
+/* ================ */
+/* ==== Device ==== */
+/* ================ */
+
+message Ports {
+  string name = 1;
+  string pciaddr = 2;
```

Comment:
Can we use an interface name instead? We're using interface names everywhere else in the product to identify interfaces

Reply:
You mean "message Interface"?

There is a "message Interface" already IIRC, but we wanted to make it clear that here we're talking about physical port settings.

Reply:
I mean `enp2s0` and etc if it makes sense